### ISSUE TITLE
Unintuitive sorting order of the latest searches when in right-handed mode.

### ISSUE THREAD

**Issue by mjdjohansson** [zobnin/aio#78]
The last "latest searches" is always added to the left of the previous search results, eventually placing it all the way to the left side of the screen. This feels a bit unintuitive when the right-handed mode is on.

![Screenshot_20230816-030740.png](https://github.com/zobnin/aio/assets/121413314/7ae9193b-7e03-4228-a264-16568f564d6a)

![Screenshot_20230816-030800.png](https://github.com/zobnin/aio/assets/121413314/2d52621b-96e7-4e12-a68e-6952e9e2d3fc)

![Screenshot_20230816-030828.png](https://github.com/zobnin/aio/assets/121413314/1ed242f7-d73b-4f75-a645-4e10c0ea6180)

![Screenshot_20230816-032357.png](https://github.com/zobnin/aio/assets/121413314/c37fa933-8929-417c-a4c9-1028fbab0b22)

**Comment by mjdjohansson** [comment]
And is also inconsistent with the sorting order of the Frequent Apps widget.

**Comment by zobnin** [comment]
Duplicate

**Comment by mjdjohansson** [comment]
I originally reported this sorting order issue in https://github.com/zobnin/aio/issues/75 after the first issue with the alignment had been solved. 

Since I never got a response about the sorting order issue and the first issue had already been closed as solved, I decided to move the second part into this new https://github.com/zobnin/aio/issues/78 issue instead. 

'Duplicate' is the first response I've gotten so far, so I'm still interested to know if you are planning to reverse the sorting order or not, or if you simply haven't decided yet?
